tests/test_mpi.py

```diff
@@ -1333,8 +1336,10 @@ def test_merge_haloupdate_if_diff_locindices_v1(self, mode):
 
         * the second and third Eqs cannot be fused in the same loop
 
-        In the IET we end up with *one* HaloSpots, placed right before the
-        second Eq. The third Eq will seamlessy find its halo up-to-date.
+        In the IET we end up with *two* HaloSpots, one placed before the
```

Comment:
does the name of the test still make sense or should it change as well?

Reply:
right, renamed